### ISSUE TITLE
tilersforums.com: detection

### ISSUE THREAD

**Issue by ecraven1993** [uBlockOrigin/uAssets#16692]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Adblock is detected. It may not detect on your first visit to the webpage, to reproduce it, I have had to sometimes click around on the forum a few times for it to show the detection. The detection displays by adding a div with id="samNotice". 

It appears the code responsible is contained inside an embedded script within the page, easily found by searching the word adblock (code too long to add here).

The only other significant reference I could find was from an externally embedded local script, [/js/user/required.js](https://www.tilersforums.com/js/user/required.js). Within this script, the function below can be found.

` loadAd: function(m) {
            var k = this;
            var l = this.$target;
            var n = k.getPositionId();
            l.attr("data-loading", true);
            XF.ajax("GET", XF.canonicalizeUrl("index.php?sam-item/" + this.options.id + "/load"), {
                position_id: n
            }, function(p) {
                if (p.ad) {
                    if (k.options.rf) {
                        var o = l.find(".samBackup, .samCloseButton, .samSupportUs").detach();
                        l.html(p.ad);
                        l.append(o)
                    } else {
                        l.prepend(p.ad)
                    }
                    if (!m) {
                        l.removeClass("samLazyLoading");
                        e(i).trigger("samInitAdblockDetection")
                    }
                    if (k.options.rf && !k.refreshInterval) {
                        k.refreshInterval = setInterval(function() {
                            k.loadAd(true)
                        }, k.options.rf)
                    }
                }
            }, {
                skipDefault: true,
                global: false
            })`

I hope the information I've attempted to dig up is helpful in fixing this adblock detection.

### URL(s) where the issue occurs.

`https://www.tilersforums.com/`

### Screenshot(s)

![image](https://user-images.githubusercontent.com/77345363/218116809-d05e5579-211a-416d-9fe3-7f4fe8296b32.png)

### uBO version

1.46.0

### Browser name and version

Chrome

### Settings

As far as I know, I am using the default settings, filters, etc.

### Notes

Please let me know if additional information is required. I did my best to provide the info that I could in a good faith effort to do as much as I am capable of before submitting this issue and request.

**Comment by mapx-** [comment]
Test

`||tilersforums.com/js/xf/$script,1p`
